packages/block-editor/src/components/iframe/style.scss

```diff
@@ -9,9 +9,10 @@
 }
 
 .block-editor-iframe__scale-container.is-zoomed-out {
-	$container-width: var(--wp-block-editor-iframe-zoom-out-container-width, 100vw);
 	$prev-container-width: var(--wp-block-editor-iframe-zoom-out-prev-container-width, 100vw);
 	width: $prev-container-width;
-	// This is to offset the movement of the iframe when we open sidebars
-	margin-left: calc(-1 * (#{$prev-container-width} - #{$container-width}) / 2);
+	// Position the iframe so that it is always aligned with the right side so that
+	// the scrollbar is always visible on the right side
+	position: absolute;
+	right: 0;
```

Comment:
Affix the iframe container to the right side of the container so its scrollbar is stuck to the right side.

Reply:
Will this work well with RTL writing direction?

Reply:
would adding `/*rtl:ignore*/` before line 17 be enough?

Reply:
I think @draganescu already tested it in an RTL environment and [said it's looking fine](https://github.com/WordPress/gutenberg/pull/66034#issuecomment-2407011122) :)